ProcessLib/LIE/SmallDeformation/CreateSmallDeformationProcess.cpp

```diff
@@ -142,14 +144,16 @@ createSmallDeformationProcess(
     else
     {
         OGS_FATAL(
-            "Cannot construct fracture constitutive relation of given type \'%s\'.",
+            "Cannot construct fracture constitutive relation of given type "
+            "\'%s\'.",
             frac_type.c_str());
     }
 
     // Fracture properties
     std::vector<std::unique_ptr<FractureProperty>> vec_fracture_property;
     //! \ogs_file_param{prj__processes__process__SMALL_DEFORMATION_WITH_LIE__fracture_properties}
```

Comment:
After code formatting, the documentation line, 154 is forgotten to move after line 155.

Reply:
Thanks! Fixing in https://github.com/ufz/ogs/pull/1999/commits/22d3fa5d6567af34dd6aa76743e53a71cef203c9 ∈ [SmallFixes](https://github.com/ufz/ogs/pull/1999)